docs/contributing/Target Framework Strategy.md

```diff
@@ -110,3 +110,22 @@ When the .NET SDK RTMs and Roslyn adopts it all occurrences of `$(NetRoslynNext)
 
 **DO NOT** include both `$(NetRoslyn)` and `$(NetRoslynNext)` in the same project unless there is a very specific reason that both tests are adding value. The most common case is that the runtime has changed behavior and we simply need to update our baselines to match it. Adding extra TFMs for this just increases test time for very little gain.
 
+## Checklist for updating TFMs (once a year)
+
+- Update `TargetFrameworks.props`.
+- Might need updating `MicrosoftNetCompilersToolsetVersion` in `eng\Versions.props` to consume latest compiler features.
+- Change `$(NetRoslynNext)` references to `$(NetRoslyn)` in project files.
+- Update TFMs in:
```

Comment:
Usually the way I approach this is I search for say `net7.0` using VSCode and exclude .cs,.vb. That typically gives the right set of files to update.

Reply:
That's what I did as well. I will update the docs to just say that, thanks.